src/main/java/com/minecolonies/coremod/client/render/RenderBipedCitizen.java

```diff
@@ -69,6 +70,34 @@ public void doRender(@NotNull final EntityCitizen citizen, final double d, final
         super.doRender(citizen, d, d1, d2, f, f1);
     }
 
+    @Override
+    protected void renderLivingAt(final EntityCitizen entityLivingBaseIn, final double x, final double y, final double z)
+    {
+        if (entityLivingBaseIn.isEntityAlive() && entityLivingBaseIn.isAsleep())
+        {
+            super.renderLivingAt(entityLivingBaseIn, x + (double)entityLivingBaseIn.getRenderOffsetX(), y, z + (double)entityLivingBaseIn.getRenderOffsetZ());
+        }
+        else
+        {
+            super.renderLivingAt(entityLivingBaseIn, x, y, z);
+        }
+    }
+
+    @Override
+    protected void applyRotations(final EntityCitizen entityLiving, final float p_77043_2_, final float rotationYaw, final float partialTicks)
+    {
+        if (entityLiving.isEntityAlive() && entityLiving.isAsleep())
+        {
+            GlStateManager.rotate(entityLiving.getBedOrientationInDegrees(), 0.0F, 1.0F, 0.0F);
+            GlStateManager.rotate(this.getDeathMaxRotation(entityLiving), 0.0F, 0.0F, 1.0F);
+            GlStateManager.rotate(270.0F, 0.0F, 1.0F, 0.0F);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 270.0F to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)